datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/http.py

```diff
@@ -0,0 +1,62 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+import click
+from datadog_checks.dev.tooling.utils import get_valid_integrations, get_check_files, get_config_spec
+
+from ..console import CONTEXT_SETTINGS, echo_info, echo_warning
+
+REQUEST_LIBRARY_FUNCTIONS = {
+    'requests.get',
+    'requests.post',
+    'requests.head',
+    'requests.put',
+    'requests.patch',
+    'requests.delete'
+}
+
+
+def validate_config_http(file, check):
+    config_valid = True
+    if os.path.isfile(file):
+        with open(file, "r") as f:
+            if 'instances/http' not in f.read():
+                echo_warning(f'Detected {check}\'s spec.yaml file does not contain `instances/http` '
```

Comment:
```suggestion
                echo_failure(f'Detected {check}\'s spec.yaml file does not contain `instances/http` '
```
 We should ensure this is fixed so let's raise the message to error

Reply:
Sounds good, will make warnings into failures.